packages/commonwealth/client/scripts/state/api/profiles/searchProfiles.ts

```diff
@@ -27,6 +27,7 @@ interface SearchProfilesProps {
   includeRoles: boolean;
   includeMembershipTypes?: 'in-group' | `in-group:${string}` | 'not-in-group';
   includeGroupIds?: boolean;
+  includeCount?: boolean;
```

Comment:
should this be true by default? Seems like its used in a lot of places

Reply:
This should be false by default I think. It is only used in the search page, but not the search bar.